framework/src/Volo.Abp.AspNetCore.Authentication.OpenIdConnect/Microsoft/Extensions/DependencyInjection/AbpOpenIdConnectExtensions.cs

```diff
@@ -39,16 +37,7 @@ public static AuthenticationBuilder AddAbpOpenIdConnect(this AuthenticationBuild
                 return authorizationCodeReceived.Invoke(receivedContext);
             };
 
-            options.Events.OnRemoteFailure = remoteFailureContext =>
-            {
-                if (remoteFailureContext.Failure is OpenIdConnectProtocolException &&
-                    remoteFailureContext.Failure.Message.Contains("access_denied"))
-                {
-                    remoteFailureContext.HandleResponse();
-                    remoteFailureContext.Response.Redirect($"{remoteFailureContext.Request.PathBase}/");
-                }
-                return Task.CompletedTask;
-            };
```

Comment:
hi

Why you remove these lines?

Reply:
Hi,
I replaced with this one they do same thing.

```c#
options.AccessDeniedPath = "/";
``` 
You can check from this
[RemoteAuthenticationHandler.cs](https://github.com/dotnet/aspnetcore/blob/main/src/Security/Authentication/Core/src/RemoteAuthenticationHandler.cs)

Reply:
Thanks for your explain.